qemu/tests/virtio_port_login_during_data_transfer.py

```diff
@@ -0,0 +1,100 @@
+import time
+import logging
+import threading
+import random
+from collections import deque
+
+from virttest import funcatexit
+from virttest import qemu_virtio_port
+from virttest.utils_virtio_port import VirtioPortTest
+
+from qemu.tests.virtio_port_login import ConsoleLoginTest
+
+EXIT_EVENT = threading.Event()
+
+
+def __set_exit_event():
+    """
+    Sets global EXIT_EVENT
+    :note: Used in cleanup by funcatexit
+    """
+    logging.info("Executing __set_exit_event()")
+    EXIT_EVENT.set()
+
+
+def run(test, params, env):
+    """
+    KVM virtio_console test
+
+    1. boot guest with virtio-serial port and virtio-console
+    2. login the guest via virtio-console
+    3. transfer data via virtio-serial, during data transfer logout and re-login
+
+    :param test: kvm test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+
+    virtio_test = VirtioPortTest(test, env, params)
+    vm, guest_worker = virtio_test.get_vm_with_worker()
+    console = virtio_test.get_virtio_ports(vm)[0][0]
+    serialports = virtio_test.get_virtio_ports(vm)[1]
+    send_port = serialports[0]
+    recv_port = serialports[1]
+    params["login_console"] = console
+    login_test = ConsoleLoginTest(test, env, params)
```

Comment:
From my checking,  only method `console_login` in `ConsoleLoginTest` is used,  then why not use `vm.wait_for_serial_login` directly ?

Reply:
@yanan-fu They are different, wait_for_serial_login utilize default serial device, currently is `isa-serial`(plan to change it to virtserialport later), but here console login utilize `virtconsole`, which follow our test case.

Reply:
Not exactly for my understanding,  For `wait_for_serial_login`, you can use param `port` to control the login method, this is what `ConsoleLoginTest.console_login` do.

Reply:
@vivianQizhu  Miss this comment ?  or different opinion ?

Reply:
@yanan-fu Oops, sorry I have missed it, you are right, let me update it.

Reply:
@yanan-fu Checked the implementation of `wait_for_serial_login()` in `virt_vm.py`, the parameter `port(virtio)` is not used, so here I should use `login_test.serial_login()`, which act the way we expect. I will test if it works right before update.

Reply:
@vivianQizhu  bad, if so, the implementation of `ConsoleLoginTest.console_login` is totally wrong.
And if use `ConsoleLoginTest.serial_login`,  pass a console  port to serial_login ?  it is the expect way ?